qemu/tests/win_virtio_driver_update_by_installer.py

```diff
@@ -92,6 +108,19 @@ def change_virtio_media(cdrom_virtio):
                                                  gagent_install_cmd,
                                                  gagent_pkg_info_cmd)
 
+    if require_winfsp:
```

Comment:
I think we don't need so many conditions.
We can run  win_driver_installer_test.viofs_svc_run(test, params, session) directly, in this function, we can check if winfsp is installed and if service is started, if not we can install winfp,register service, start service.